addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
         if (wait > 0) {
```

Comment:
Don't we want to check if the refresh token has expired rather than if the authentication token has expired? That is my understanding of refresh tokens, they out last the authentication tokens.